RemoveNthNodeFromEndOfList.js

```diff
@@ -0,0 +1,35 @@
+var removeNthFromEnd = function (head, n) {
```

Comment:
1. First take a curr node as Head.
2. Increment the curr node to its next by n times
3. Now increment both curr and head simulatenously one by one until, curr->next is not null.
4. Finally when the curr's next is null, your head is the answer.

Reply:
https://leetcode.com/submissions/detail/810291724/